### ISSUE TITLE
Disallow dangrous parameters in the config file

### ISSUE THREAD

**Issue by mariocynicys** [talaia-labs/rust-teos#217]
Right now, the parameters `overwrite_key` (overwrites\* the tower's key and gives it a new identity) and `force_update` (introduced in #216) are allowed to be set in the configuration file.

Since they are more of a fail-safe and rather dangerous commands, we should assert that the user doesn't set them in the config file and only settable as command line flags.

**Comment by sr-gi** [comment]
Concept ACK

**Comment by anipaul2** [comment]
> Right now, the parameters `overwrite_key` (overwrites* the tower's key and gives it a new identity) and `force_update` (introduced in #216) are allowed to be set in the configuration file.
> 
> Since they are more of a fail-safe and rather dangerous commands, we should assert that the user doesn't set them in the config file and only settable as command line flags.

So, removing the command from the config file and allowing the user to use it only in cli?

**Comment by mariocynicys** [comment]
> So, removing the command from the config file and allowing the user to use it only in cli?

Yeah something like that. You might only disallow it by sanity checking that it's not set. Not that we have to remove it from the `Config` struct.

**Comment by mariocynicys** [comment]
Just noticed there is actually nothing dangerous here because `Config::patch_with_options` completely ignores the config file value of `overwrite_key` & `force_update` and uses the command line options instead (which default to false).

I think we can just note that in a comment in the template config file so that users don't expect something when setting these ignored parameters to `true`.

Also, write a comment about it in https://github.com/talaia-labs/rust-teos/blob/1a89c5da70278ac5019a3bbb505b1923648a43da/teos/src/config.rs#L206-L207 since we all missed it xD.
